programs/server/Server.cpp

```diff
@@ -1970,7 +1970,8 @@ catch (...)
 {
     /// Poco does not provide stacktrace.
     tryLogCurrentException("Application");
-    throw;
+    auto code = getCurrentExceptionCode();
+    return code ? code : -1;
```

Comment:
This doesn't seem right. code is an int, so multiples of 256 are != 0 and the error code won't be changed, right?

```
catch (...)
{
    /// Poco does not provide stacktrace.
    tryLogCurrentException("Application");
    auto code = 256;
    return code ? code : -1;
}
```
Exits the program with 0 still